TEAMS_ROLES_STRUCTURES.md

```diff
@@ -230,13 +233,14 @@ Develop the Golang implementation of the IPFS Protocol, go-ipfs.
 - Upgrade and build dev tools (i.e gx).
 - Set the guidelines for writing, maintaining and managing the Go codebases, test suites, and contributions.
 
-#### IPFS GUI
+#### IPFS GUI/UX
```

Comment:
@ipfs/wg-gui-ux given current communications in written form and previous conversations, my understanding is that the GUI Working Group now uses GUI/UX and GUI interchangeably given the scope increase. Please confirm if this is accurate and let me know if not for me to change.